theforeman.org/scripts/debian/build_project.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
what is this file for?

Reply:
Absolutely nothing. Was accidental add from when I thought about trying to create a consolidated build script that combined all the ways to build source and then realized how crazy it is.